python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -1279,10 +1279,10 @@ def my_dbt_asset(dbt: DbtCliResource):
                     dbt.cli(["run-operation", "my-macro", json.dumps(dbt_macro_args)]).wait()
 
 
-                @op
+                @op(out={})
```

Comment:
Generally, ops should have outputs so that other ops can depend on them.

Reply:
What do you suggest that the output should be? And what about this case where it is a standalone op?

I'm happy to clarify in the API docs that this is an example of a standalone op if that makes things more clear.

Reply:
> What do you suggest that the output should be? 

Its type can/should just be `None`

Reply:
That doesn't work. We are yielding other events, so the return type cannot be `None`. It would cause a type-checking error.

Reply:
It could be `@op(out=Out(None))`

We might also need to `yield Output(None)` from within the body.

Btw I realize this is quite awkward, which is unfortunate, but I do think running ops downstream will be a primary use case for people doing this.

Reply:
I got it.. very painful, but it makes sense.

I added it as a separate example and added a unit test.

Reply:
Having these two similar examples here feels a little messy.

I played around a bit and it looks like this executes successfully:

```python
from dagster import AssetMaterialization, Nothing, Out, op


@op(out=Out(Nothing))
def op1():
    yield AssetMaterialization(asset_key="asset1")
```

Thoughts on going with either `out=Out(Nothing)` or `out=Out(type(None))` and having a single example.

Reply:
I didn't know you could do that. Nice. Updated the example.